bitnami/cassandra/templates/_helpers.tpl

```diff
@@ -65,27 +65,39 @@ Also, we can't use a single if because lazy evaluation is not an option
 {{- end -}}
 {{- end -}}
 
-
 {{/*
 Return the proper metrics image name
 */}}
-{{- define "cassandra.metrics.image" -}}
-{{- $registryName :=  .Values.metrics.image.registry -}}
+{{- define "metrics.image" -}}
```

Comment:
it is a good practice to use namespace for defined templates

https://helm.sh/docs/chart_best_practices/#names-of-defined-templates